### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by jtpio** [jupyterlab/retrolab#84] [General]
Dumping the first iteration on the logo here for reference, and so it is easy to find later too:

### Social

![jupyterlab-classic-social](https://user-images.githubusercontent.com/591645/105419584-3bbeff00-5c3f-11eb-8aa9-b9323064ea85.png)

### PNG

![jupyterlab-classic](https://user-images.githubusercontent.com/591645/105419599-3feb1c80-5c3f-11eb-9dff-379cd7f4fbc4.png)

**Comment by yuvipanda** [comment]
Nice! Is there a square version as well? I'm also curious how to handle the extra height in toolbars and stuff.

**Reply by jtpio** [reply]
> Is there a square version as well?

Not yet, but it would make sense to make it square by default.

Also since it's quite big we might still want to use the default Jupyter logos and icons in toolbar for now (unless someone comes up with a nice one for classic). It's mostly used in the readme and the about popup for now:

![image](https://user-images.githubusercontent.com/591645/105473501-7bb4cf00-5c9d-11eb-8ba8-f530d5b2f410.png)
